website_and_docs/content/documentation/webdriver/bidirectional/_index.en.md

```diff
@@ -19,8 +19,7 @@ The traditional WebDriver model of strict request/response commands will be supp
 with the ability to stream events from the user agent to the controlling software via WebSockets,
 better matching the evented nature of the browser DOM.
 
-Because it's a bad idea to tie your tests to a specific version of a specific browser,
-the Selenium project recommends using WebDriver BiDi wherever possible.
-However, until the spec is complete there are many useful things that the CDP offers.
-To help keep your tests independent and portable, Selenium offers some useful helper classes.
-At the moment, these use the CDP, but when we shall be using WebDriver Bidi as soon as possible
+As it is not a good idea to tie your tests to a specific version of any browser, the
+Selenium project recommends using WebDriver BiDi wherever possible.
+
+However, until the specification is complete there are many useful things that CDP (Chrome DevTools Protocol) offers. To help keep your tests independent and portable, Selenium offers some useful helper classes as well. At the moment, they use the CDP, but soon it could be done using WebDriver BiDi.
```

Comment:
```suggestion
As it is not a good idea to tie your tests to a specific version of any browser, the
Selenium project recommends using WebDriver BiDi wherever possible.

However, until the specification is complete there are many useful things that 
CDP (Chrome DevTools Protocol) offers. To help keep your tests independent 
and portable, Selenium offers some useful helper classes as well. At the 
moment, they use the CDP, but soon it could be done using WebDriver BiDi.
```